src/Umbraco.Web.UI.Client/src/views/propertyeditors/grid/dialogs/config.controller.js

```diff
@@ -16,7 +16,23 @@ function ConfigController($scope) {
             $scope.model.close();
         }
     }
-    
+
+    vm.showEmptyState = function() {
+        return $scope.model.config.length === 0
+            && Object.keys($scope.model.config).length === 0
+            && $scope.model.styles.length === 0;
+    }
```

Comment:
@lars-erik I guess this could be defined as:

```
vm.showEmptyState = function() {
        return vm.showConfig() === false && vm.showStyles() === false;
}
```

Reply:
For sure! I'll see about it suddenly.